### PR TITLE
pacific: backport mgr/nfs bits 

### PR DESCRIPTION
**Pull request by liewegas**
- #43611 mgr/nfs fixes for rgw user exports
- #43709 test_nfs fix to add a wait
- #43504 mgr/nfs: nfs cluster config get
- #43743 use mgr_module tool_exec()

Blocked by
- [x] #43682

### CODE REVIEW

**Comment by epuertat** [comment]
@liewegas jfyi: this also contains the Cluster Expansion Wizard (it's not only NFS stuff).

**Comment by liewegas** [comment]
> @liewegas jfyi: this also contains the Cluster Expansion Wizard (it's not only NFS stuff).

Yeah i based it on top of #43682 because there were a few mgr/nfs patches in there

**Comment by epuertat** [comment]
jenkins test dashboard

**Comment by epuertat** [comment]
jenkins test dashboard cephadm

**Comment by epuertat** [comment]
Dashboard check failed with [xmlsec corrupted package](https://jenkins.ceph.com/job/ceph-dashboard-pull-requests/9179/consoleFull#-85362248744e9240e-b50a-4693-bac0-8a991bac86ac).

jenkins test dashboard

**Comment by epuertat** [comment]
jenkins test dashboard

**Comment by alfonsomthd** [comment]
`jenkins test dashboard` failed: https://dashboard.cypress.io/projects/k7ab29/runs/5671/overview

Re-running job...

**Comment by alfonsomthd** [comment]
jenkins test dashboard

**Comment by alfonsomthd** [comment]
@liewegas Now that #43682 has been merged, would it be better to rebase the branch?

**Comment by liewegas** [comment]
jenkins test make check

File: PendingReleaseNotes [hunk 1]
```diff
@@ -87,6 +87,21 @@
...
+  *--fsname myfs`` instead of ``ceph nfs export create cephfs
+  *mycluster /foo myfs`` to ensure correct behavior with any
+  *version.
+
+* mgr-pg_autoscaler: Autoscaler will now start out by scaling each
+  pool to have a full complements of pgs from the start and will only
+  decrease it when other pools need more pgs due to increased usage.
+  This improves out of the box performance of Ceph by allowing more PGs 
+  to be created for a given pool.
+
```
Comment (neha-ojha) [inline PendingReleaseNotes:104]: nit: not relevant but probably just crept in with the cherry-pick

**Review by neha-ojha — APPROVED** [review]
LGTM based on test runs

**Pull request merged**